Dockerfile

```diff
@@ -41,10 +44,17 @@ ENV NVIDIA_VISIBLE_DEVICES=all
 
 WORKDIR /build
 
+COPY Makefile .
+RUN make get-sources
+COPY go.mod .
+RUN make prepare
 COPY . .
 RUN make build
 
-FROM requirements
+###################################
+###################################
+
+FROM builder
```

Comment:
better don't do this - currently seems we leave some leftover, which makes  rebuild from the same host fail, See rationale in https://github.com/go-skynet/LocalAI/pull/434

Reply:
great - exactly what I was looking for, thank you!